src/content/docs/dev/send-tokens/interchain-tokens/developer-guides/link-custom-tokens-deployed-across-multiple-chains-into-interchain-tokens.mdx

```diff
@@ -300,120 +307,158 @@ main().catch((error) => {
+    "0x510e5EA32386B7C48C4DEEAC80e86859b5e2416C", //  the address of the operator
+    { value: ethers.utils.parseEther("0.001") },
   );
 
-  return gas;
+  console.log(`
+    Transaction Hash: ${deployTxData.hash},
+    salt: ${salt}`);
 }
 
 //...
 ```
 
-### Perform remote token manager deployment
+### Update `main()` to register custom token
 
-Create a `deployRemoteTokenManager()` function. This will deploy the remote Token Manager on the Polygon testnet. Make sure to change the salts to the value you saved from a previous step.
+Update `main()` to execute `registerCustomTokenOnBSC()` :
 
 ```javascript
 //...
 
-// Deploy remote token manager : Polygon
-async function deployRemoteTokenManager() {
-  // Get a signer to sign the transaction
-  const signer = await getSigner();
+async function main() {
+  const functionName = process.env.FUNCTION_NAME;
+  switch (functionName) {
+    //...
+    case "registerCustomTokenOnBSC":
+      await registerCustomTokenOnBSC();
+      break;
+    default:
+    //...
+  }
+}
 
-  // Get the InterchainTokenService contract instance
-  const interchainTokenServiceContract = await getContractInstance(
-    interchainTokenServiceContractAddress,
-    interchainTokenServiceContractABI,
-    signer,
-  );
+//...
+```
 
-  // Create params
-  const param = ethers.utils.defaultAbiCoder.encode(
-    ["bytes", "address"],
-    [signer.address, polygonCustomTokenAddress],
-  );
+### Run the `customInterchainToken.js` script to deploy to BSC
 
-  const gasAmount = await gasEstimator();
+Run the script in your terminal to register the custom token, once again specifying the `BSC` testnet (the source chain where all transactions are taking place):
 
-  // Deploy the token manager
-  const deployTxData = await interchainTokenServiceContract.deployTokenManager(
-    "0x7f5fe989334ad23b77e8da841f672394d021ae83ff0a24afb461ba19d72e3553", // change salt
-    "Polygon",
-    MINT_BURN,
-    param,
-    ethers.utils.parseEther("0.01"),
-    { value: gasAmount },
-  );
+```javascript
+FUNCTION_NAME=registerCustomTokenOnBSC npx hardhat run customInterchainToken.js --network bsc
+```
 
-  // Get the tokenId
-  const tokenId = await interchainTokenServiceContract.interchainTokenId(
-    signer.address,
-    "0x7f5fe989334ad23b77e8da841f672394d021ae83ff0a24afb461ba19d72e3553", // change salt
-  );
+You should see something similar to the following on your console:
 
-  // Get the token manager address
-  const expectedTokenManagerAddress =
-    await interchainTokenServiceContract.tokenManagerAddress(tokenId);
+```bash
+Transaction Hash: 0xf0c3fbbece17cc12d9de76182540afb4f5c6c47aeca65c40f9d58f71e362dffe,
+salt: 0xd58c9142d863dde9f0b9ea68b10cce9470626040187a00c16b7b3d4edae70cf9
+
+```
+
+## Link custom token with the Interchain Token Factory from BSC to Avalanche Fuji chain
+
+In this section, you must link the custom token using the Interchain Token Factory between the chains where your token exists.
```

Comment:
what is the difference between linking the token and the previous step of registering the token? We should explain the difference a bit I think